### PR TITLE
Update ribbon-issues-button-hidden.md

### PR DESCRIPTION
**Pull request by hazhouMSFT**
Update Troubleshooting guide on details for context based commands, and why buttons can be hidden even if it appears "true" in command checker.

### CODE REVIEW

**Comment by PRMerger9** [comment]
@hazhouMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by KumarVivek — APPROVED** [review]

File: support/power-platform/power-apps/ribbon-issues-button-hidden.md [hunk 1]
```diff
@@ -41,6 +41,9 @@ Once the Command Checker tool has been enabled, within the application in each o
 
     :::image type="content" source="media/ribbon-issues-button-hidden/new-hidden.png" alt-text="Screenshot shows the New button on the contact entity's grid page is not visible and is represented by an item labeled New (hidden).":::
 
+    > [!NOTE]
+    > If you see all rules evaluate to true, but the button is still hidden, it may be due to [context sensitive commands in grids](https://support.microsoft.com/en-us/topic/button-in-command-bar-does-not-appear-after-grid-item-selection-1100b6b5-ab64-6047-82c5-924a2b38296a). When records are selected on a grid, all buttons without a SelectionCountRule will be considered not relevant to the selected record(s) and be hidden even if their rule evaluation is true.  Note this does not effect flyouts, since flyout children might still have record based commands.
```
Comment (brflood) [inline support/power-platform/power-apps/ribbon-issues-button-hidden.md:45]: I think the URL should not contain the language locale "en-us/" part to allow the locale to automatically resolved

**Comment by PRMerger16** [comment]
@hazhouMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: support/power-platform/power-apps/ribbon-issues-button-hidden.md (hunk 1 above)
Comment (hazhouMSFT) [reply support/power-platform/power-apps/ribbon-issues-button-hidden.md]: Removed, good point.

**Comment by hazhouMSFT** [comment]
This is ready for merge, waiting for David Packman or Christy Su to merge.

**Review by brflood — APPROVED** [review]

**Comment by PRMerger10** [comment]
@lucciz01 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by ChristySu — APPROVED** [review]
Approved.

**Comment by PRMerger13** [comment]
@lucciz01 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger3** [comment]
@lucciz01 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by lucciz01** [comment]
@brflood @hazhouMSFT @KumarVivek, minor changes are made to this PR and it is merged. Feel free to let me know if any questions.

**Comment by David-Pack** [comment]
Approved

File: support/power-platform/power-apps/ribbon-issues-button-hidden.md [hunk 2]
```diff
@@ -41,6 +41,9 @@ Once the Command Checker tool has been enabled, within the application in each o
...
+ 
 1. Click the **Command Properties** tab to display the details of the command for this button. This will show the enable rules and display rules, along with the result (**True**, **False**, **Skipped**) of each rule evaluation. The following example shows the **New (hidden)** button's command to be `Mscrm.NewRecordFromGrid` and there is an enable rule named `new.contact.EnableRule.EntityRule` that has evaluated to **False**, as a result the button will be hidden.
 
     :::image type="content" source="media/ribbon-issues-button-hidden/command-properties-new-hidden.png" alt-text="Screenshot shows the Command properties details of the command for the New (hidden) button.":::
```
Comment (janderadenis) [inline support/power-platform/power-apps/ribbon-issues-button-hidden.md:49]: Fuck off

**Pull request merged**